fiftyone/core/collections.py

```diff
@@ -916,26 +916,45 @@ def _get_per_sample_frames_bytes(self):
         results = self._aggregate(pipeline=pipeline, attach_frames=True)
         return {str(r["_id"]): r["size_bytes"] for r in results}
 
+    def _get_first(self, limit=1, field="_id", reverse=False):
+        direction = -1 if reverse else 1
+        pipeline = [{"$sort": {field: direction}}, {"$limit": limit}]
+
+        return self._aggregate(
+            pipeline=pipeline, detach_frames=True, detach_groups=True
```

Comment:
`iter_samples()` includes `detach_frames=True, detach_groups=True` so we should use it here too